kafka-client/src/test/java/dev/responsive/kafka/api/ResponsiveKafkaStreamsTest.java

```diff
@@ -126,6 +127,7 @@ public void setUp() {
 
     properties.put(RESPONSIVE_ORG_CONFIG, "responsive");
     properties.put(RESPONSIVE_ENV_CONFIG, "test");
+    properties.put(ResponsiveConfig.STORAGE_BACKEND_TYPE_CONFIG, StorageBackend.CASSANDRA.name());
```

Comment:
I thought the point of this PR is to change our tests to run with Mongo?

Reply:
Some of them seemed to rely on Cassandra (eg for fault injection, like in this case) and I didn't want to block the entire PR on updating every test like this. So the ones you see where I set Cassandra explicitly were the ones that failed because they did something Cassandra specific.

Usually the test had some hook through a mock cassandra client, so we'd "just" need to switch that setup over to use mongo. I say "just" in quotes because my impression is this might be a nontrivial effort